mkdocs.yml

```diff
@@ -2,7 +2,12 @@ site_name: Kubernetes Gateway API Inference Extension
 repo_url: https://github.com/kubernetes-sigs/gateway-api-inference-extension
 repo_name: kubernetes-sigs/gateway-api-inference-extension
 site_dir: site
+site_url: 'https://gateway-api-inference-extension.sigs.k8s.io/'
 docs_dir: site-src
+extra:
+  version:
+    provider: mike
```

Comment:
@artberger does `mike` also need to be added to the `plugins` list?

Reply:
@danehans it can be if we want to change any of the default `mike` configuration values: https://github.com/jimporter/mike?tab=readme-ov-file#configuration